engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-repository-sets.html

```diff
@@ -16,6 +16,18 @@
            ng-model="repositorySetFilter"/>
   </div>
 
+  <div data-block="filters">
+    <label class="checkbox-inline" title="{{ 'Show all Repository Sets available to this Activation Key' | translate }}">
+      <input type="checkbox" ng-model="contentAccessModes.contentAccessModeAll" ng-change="toggleFilters()"/>
+      <span translate>Show All</span>
```

Comment:
"Show all Repository Sets currently available for Organization, not just those available for this activation key through its subscriptions."

How about that? That will match the info tip always displayed on page.